dspace-api/src/main/java/org/dspace/content/packager/RoleDisseminator.java

```diff
@@ -461,6 +463,13 @@ protected void writeEPerson(EPerson eperson, XMLStreamWriter writer,
             writer.writeEmptyElement(SELF_REGISTERED);
         }
 
+        if (null != eperson.getWelcomeInfo()) {
```

Comment:
Objects.nonNull

Reply:
I used null != because I wanted to keep style of the class. So should I still change it?